.travis.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  - docker
+
+notifications:
+  email:
+    recipients:
+      - cc@monexa.co.uk
+    on_success: always
+    on_failure: always
+
+before_install:
+  - docker --version
```

Comment:
I think we can probably get rid of this before_install and the after_success now that we know they're working?